pkg/cover/store.go

```diff
@@ -153,8 +167,32 @@ func (l *fileStore) load() error {
 	return nil
 }
 
-func (l *fileStore) Set(services map[string][]string) {
-	panic("TO BE IMPLEMENTED")
+func (l *fileStore) Set(services map[string][]string) error {
+	l.mu.Lock()
+	defer l.mu.Unlock()
+
+	// no error will return from memorystore.set
+	_ = l.memoryStore.Set(services)
```

Comment:
I think `ignore err` is not a goole practice since probably the behavior of `memoryStore.Set` may be changed in the future.